frontend/pages/specz_catalogs.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react'
+import { Typography } from '@mui/material'
+
+function SpeczCatalogs() {
+  return (
+    <div style={{ textAlign: 'center' }}>
```

Comment:
Okay, I made the suggested changes.
I put 'div' because it is a temporary screen, as I am already developing the main screen. Thank you for the tips.